src/StringUtil.php

```diff
@@ -6,6 +6,10 @@
 
 class StringUtil
 {
+    public const WINDOWS_NEW_LINE = "\r\n";
+
+    public const TAB = "\t";
```

Comment:
Finde ich unnötig indirekt. Das Wissen wie Escape-Sequenzen für Strings in PHP funktionieren brauchen wir meiner Meinung nach nicht in einer Utility-Klasse in diesem Projekt kodieren.